DQM/HLTEvF/plugins/LumiMonitor.cc

```diff
@@ -31,12 +30,12 @@ namespace {
   };
 
   struct Histograms {
-    ConcurrentMonitorElement numberOfPixelClustersVsLS;
-    ConcurrentMonitorElement numberOfPixelClustersVsLumi;
-    ConcurrentMonitorElement lumiVsLS;
-    ConcurrentMonitorElement puVsLS;
-    ConcurrentMonitorElement pixelLumiVsLS;
-    ConcurrentMonitorElement pixelLumiVsLumi;
+    dqm::reco::MonitorElement const* numberOfPixelClustersVsLS;
```

Comment:
what is the `dqm::reco` namespace ?

Reply:
Introduced here, with explanation: https://github.com/cms-sw/cmssw/pull/27476

It might not be strictly required to use the fully qualified name everywhere, but in the global analyzers you will see them more than usual (since the declarations are outside the class scope).